docs/user-guide/styling-your-docs.md

```diff
@@ -349,6 +355,24 @@ Now the generated pages will include links to the template provided libraries as
 well as the library included in the `custom_dir`. The same would be required for
 any additional CSS files included in the `custom_dir`.
 
+## Localizing a theme
```

Comment:
This section makes sense here as it covers how to modify an existing theme. However, we also need to document things for those who are building a theme from scratch. At a minimum, we need to list locale files along the [types of files][1] that a theme may contain and we should document the `locale` [theme config option][2].

Of course, developers of third party themes can use whatever tools they want to manage their translations, but they do need to know what MkDocs expects. For example, they need to be informed that any theme files under the `locale` subdirectory are ignored by MkDocs for anything except translation and that that same subdirectory is the only location MkDocs looks for translation files and that those files must be MO files.

[1]: https://www.mkdocs.org/user-guide/custom-themes/#theme-files
[2]: https://www.mkdocs.org/user-guide/custom-themes/#theme-configuration